protocol/avdl/stellar1/local.avdl

```diff
@@ -40,21 +40,29 @@ protocol local {
     UNKNOWN_5
   }
 
+  enum ParticipantType {
+    NONE_0,
+    KEYBASE_1,
+    STELLAR_2,
+    SBS_3    
+  }
+
   record PaymentLocal {
     TransactionID id;               // For relay payments, the funding tx.
     TimeMs time;
     PaymentStatus statusSimplified; // we are working on making this correct for relay payments...
     string statusDescription;       // "pending", "completed", "error"
     string statusDetail;
     string amountDescription;       // "+ 1,323.1234567 XLM"
+    string amountValue;             // "1,323.1234567 XLM" (no + or -)
```

Comment:
This is too much. Can we just have `amountDescription` without the sign and gui can display + or - or neither based on `delta`? @buoyad

Reply:
I guess that's not really my objection. It's that we use the type `string` for actual machine readable amounts and the word 'value' sounds very canonical to me. If yall are into the doubled representation then how about `amountDescription` and `amountDescriptionSigned` or something?

Reply:
`amountDescription` sans sign is fine, we can render the + or -.